File: assignment15.py

vars = [9, 2]

def modulusFunc(vars):
    temp_var = int(vars[0] / vars[1])
    temp_var_2 = vars[0]
    for i in range(1, temp_var+1):
        temp_var_2 -= vars[1]
    print(temp_var_2)

modulusFunc(vars)

File: javascript_assignments/assignment15.js
let vars = [9, 2]

function modulusFunc(vars){
    temp_var = vars[0]/vars[1]
    temp_var_2 = vars[0]
    
    for (let i = 0; i < 1+temp_var_2; i++){
        temp_var_2 -= vars[1]
    }

    console.log(temp_var_2)
}

modulusFunc(vars)